hypothesis-python/src/hypothesis/statistics.py

```diff
@@ -83,9 +83,11 @@ def describe_statistics(stats_dict):
         statuses = Counter(t["status"] for t in cases)
         runtime_ms = format_ms(t["runtime"] for t in cases)
         drawtime_ms = format_ms(t["drawtime"] for t in cases)
+        gctime_ms = format_ms(t["gctime"] for t in cases)
+        gctime_txt = "" if gctime_ms[0] == "<" else f"and {gctime_ms} in gc"
```

Comment:
Noting that this will almost never print anything, since GC is infrequent-but-slow. The 95th percentile is perhaps not the best measure?